README.md

```diff
@@ -72,13 +74,31 @@ Both Docker and Kubernetes require specialized staff to manage. Via dyrector.io,
 
 Seamless testing whenever your team wants to test the application, without waiting for a SysAdmin to set up an environment.
 
-## Quick Start
+## Get Started
+
+dyrectorio CLI, is a command-line interface that lets you run and manage the whole dyrectorio projects locally.
+
+This means you get a full-featured dyrectorio platform running locally on your machine.
 
 ### Prerequirements
 
--   Install Docker and Docker Compose to your machine
+- Docker installed on your system but Podman works, too. In case of Linux, you need Docker Compose, as well.
+- Go Compiler to run the CLI by its source code.
```

Comment:
it's

Reply:
"Go Compiler to run the CLI by _it is_ source code." sounds wrong to me 😅

Reply:
@m8vago refer to the apostrophe

Reply:
Yes I know, but this way the sentence means what I've quoted, so I think the original one was the correct one.

Reply:
@m8vago ?

Reply:
@robot9706 is right. sorry. He is the true master of "angology"